pulp_rpm/app/migrations/0047_modulemd_datefield.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 3.2.16 on 2022-11-08 19:12
+
+from django.db import migrations, models
+
+import yaml
+
+
+def parse_date(apps, schema_editor):
+    """Parse date from snippet."""
+    ModulemdObsolete = apps.get_model("rpm", "ModulemdObsolete")
+
+    obsoletes_to_update = []
+
+    for obsolete in ModulemdObsolete.objects.all():
+        parsed_snippet = yaml.safe_load(obsolete.snippet)
+        obsolete.modified = parsed_snippet['data']['modified']
+        obsoletes_to_update.append(obsolete)
+
+    ModulemdObsolete.objects.bulk_update(obsoletes_to_update, ["modified"])
+
+
+class Migration(migrations.Migration):
```

Comment:
I know we had a reason for doing this (I remember that we discussed it) but can you refresh on what the reason was?  IIRC we went back and forth a couple of times on whether it was better to use a date field or to just shove it into a text field.

Reply:
Ack, reasonable.  At least the format is lexicographically sortable.